client/src/components/Lists/PitchTeams/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC, ReactElement } from 'react';
+import { IPitch, IUser } from 'ssw-common';
+
+import { FieldTag } from '../..';
+import { useTeams } from '../../../contexts';
+
+interface PitchTeamsProps {
+  pitch: IPitch;
+  user?: Pick<IUser, '_id'>;
```

Comment:
Same comment about the typing of this with regards to what I said about Pitch Interests. It doesn't make sense to combine UserTeams and PitchTeams though because this definitely has unique Pitch stuff that is nice to have in a component.